appveyor-nightly.yml

```diff
@@ -61,8 +65,6 @@ test_script:
 after_test:
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, ubuntu18]"
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, centOS7]"
-  - rake "download_portable[develop]"
```

Comment:
Should this also stay? It's new on Develop too for creating a standalone OSPSuite-R?